go.mod

```diff
@@ -44,9 +43,33 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0-20170531160350-a96e63847dc3
 	k8s.io/api v0.15.11
-	k8s.io/apiextensions-apiserver v0.0.0-20200318010201-8546efc3bc75 // kubernetes-1.15.11
+	k8s.io/apiextensions-apiserver v0.0.0 // kubernetes-1.15.11
```

Comment:
This one version is non existent. And could lead to a problem.

Reply:
good catch